def minha_funcao():
    x = 10
    print("Valor de x dentro da função:", x)

minha_funcao()


print("Valor de x fora da função:", x)  # Tentando acessar a variável x fora da função (irá gerar um erro)
